app/views/sharing_mailer/shared_work_package.html.erb

```diff
@@ -0,0 +1,37 @@
+<table>
+  <tr>
+    <%= placeholder_cell('20px', vertical: false) %>
+  </tr>
+</table>
+
+<table <%= placeholder_table_styles(width: '100%', style: 'width:100%;max-width:700px') %>>
+  <tr>
+    <%= placeholder_cell('12px', vertical: true) %>
+    <td>
+      <%= render partial: 'sharing_mailer/notification_mailer_header',
```

Comment:
Hi @aaron-contreras 

I'd like to avoid having to add all these extra templates (mailer_body, mailer_header and notification_row) as we have all of them already and its hard to maintain multiple ones.
I am aware that the current Figma mockups define the mails slightly different, so I talked to the product team again and they agreed, that the emails should look exactly like out existing ones. So there is no need to add additional templates anymore but you can just use the exisiting ones instead.